scripts/check_frontend_coverage.py

```diff
@@ -328,6 +328,8 @@
     'schema-based-list-editor.directive.ts',
     'schema-based-list-viewer.directive.ts',
     'schema-based-primitive-viewer.directive.ts',
+    'schema-based-subtitled-html-editor.directive.ts',
```

Comment:
I don't think we should be adding files to this list (as mentioned in the comment above). Please write tests for these 2 files or provide some reasoning on why they need to be added here.

Reply:
The two files schema-based-subtitled-html-editor.directive.ts',  'schema-based-subtitled-unicode-editor.directive.ts' are almost identical to 'schema-based-html-editor.directive.ts' and 'schema-based-unicode-editor.directive.ts', and those are currently not tested. Should I write tests for all 4 files? Or keep as is for now?